src/repo/lock.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const path = require('path')
+const debug = require('debug')
+const { lock } = require('proper-lockfile')
+
+const log = debug('repo-migrations:repo_fs_lock')
+const lockFile = 'repo.lock'
+
+/**
+ * Duration in milliseconds in which the lock is considered stale
+ * @see https://github.com/moxystudio/node-proper-lockfile#lockfile-options
+ * The default value of 10000 was too low for ipfs and sometimes `proper-lockfile`
+ * would throw an exception because it couldn't update the lock file mtime within
+ * the 10s threshold. @see https://github.com/ipfs/js-ipfs-repo/pull/182
+ * Increasing to 20s is a temporary fix a permanent fix should be implemented in
+ * the future.
```

Comment:
I believe this problem has been resolved so no need to alter this stale timeout.

Reply:
Well, this was directly taken from https://github.com/ipfs/js-ipfs-repo/blob/master/src/lock.js so I am a bit confused. It is still there...

Should I remove this custom STALE_TIME?

Reply:
I think that is also out of date - it was fixed in proper-lockfile here I believe https://github.com/moxystudio/node-proper-lockfile/pull/74